--- mallodroid.py	2017-07-25 17:28:29.275727268 +0100
+++ mallodroid-patched.py	2017-07-25 17:28:52.359918492 +0100
@@ -401,7 +401,7 @@
 	_vm = dvm.DalvikVMFormat(_a.get_dex())
 	_vmx = uVMAnalysis(_vm)
 	
-	if 'INTERNET' in _vmx.get_permissions([]):
+	if 'android.permission.INTERNET' in _vmx.get_permissions([]):
 		print "App requires INTERNET permission. Continue analysis..."
 		
 		_vm.create_python_export()
@@ -427,4 +427,4 @@
 		print "App does not require INTERNET permission. No need to worry about SSL misuse... Abort!"
 
 if __name__ == "__main__":
-	main()
\ No newline at end of file
+	main()
